src/build/help/help.xml

```diff
@@ -1057,6 +1057,18 @@
                         <example>pg-backup</example>
                     </config-key>
 
+                    <config-key id="repo-sftp-known-hosts" name="SFTP known hosts file">
+                        <summary>a known hosts file to search.</summary>
+
```

Comment:
"Known hosts file to search." is fine here.

Reply:
Updates in
https://github.com/pgbackrest/pgbackrest/pull/2125/commits/4fd017c80716d71a7ffff1f25379b9da196f7001
https://github.com/pgbackrest/pgbackrest/pull/2125/commits/6811405f150c8f1095c3da31d952f0ddebc58be1
https://github.com/pgbackrest/pgbackrest/pull/2125/commits/a3a82046feab0a915e0c116f867818b86ae42621